### DISCUSSION TITLE
Textual Inversion with Kohya GUI

### DISCUSSION THREAD

**Discussion by Woisek** [bmaltais/kohya_ss#326] [Q&A]
Now that we got a great tutorial on making LoRAs with this great tool, I'm missing an equal great tutorial on how to make Textual Inversion models with this.
Is it possible to get one, with all those settings explained like in the LoRA tut? Would help a ton. :)

**Comment by quasiblob** [comment]
Not sure if you already know about TI training, but most of the settings look pretty similar to what you get in Automatic1111's web ui, so read their wiki (there is a TI training page) or watch some Stable Diffusion tutorials by some YouTubers? 

Anyway, despite of what I said, I too would like to see a tutorial that explains the specifics related to Kohya gui implementation of TI training.

**Reply by Woisek** [reply]
Yeah, I have noticed the similarity and I did some TIs with it, but then ... there are much more settings on Kohyas side that make me think we can create better TIs here then in WebUI. So I would love to see such an indepth tutorial for TI like he did with LoRA. :)

**Reply by bmaltais** [reply]
I did play with TI a bit and did get some good results... but I mostly focus on LoRAs at the moment. With the new Optimizer and all there is potential for improved TIs under kohya...

Maybe I will get to play more with that one day.

**Reply by Woisek** [reply]
Pleeeease, play soooon ... :) 👍

**Reply by Woisek** [reply]
Any news on the tutorial? I really like to make TI in the Kohya_ss GUI. 🙏😃

**Reply by andupotorac** [reply]
Would be good to have one for creating TI for negative prompts too.

**Reply by misterwackydoodle** [reply]
The default learning rate for TIs is too low.  They seem to need higher.  I'm getting better results with 0.0003.  Have problem getting TI to resume after a pause.  The best results I obtained with TI on automatic1111, I did resume 3 or 4 times, cutting the learning rate in half about each time.  Not sure if TI resumed correctly on kohya gui, though it seems Lora training does (message about 'resuming from ...' and iterations continue where they left off).  This would be nice to have as a builtin feature.  Anyway having fun with this so far, thanks!
